### DISCUSSION TITLE
TodoApp tasks is unscrollable.

### DISCUSSION THREAD

**Discussion by applicatlat** [flet-dev/flet#1991] [Q&A]
Since I have sent you a detailed explanation regarding an UI bug with a detailed code snippet and you asked me to shorten it, I shortened the this code as well due that response. Also I would like to remind you that you did not look at that code and closed it without and explanation. Here is the link of it:
https://github.com/flet-dev/flet/discussions/1365

Current issue:
The TodoApp self.tasks = ft.Column() that you shared on the flet.dev does not have scroll funcion. Since you recommended it to add self.expand on a listview project, I did the same and added the self.tasks = ft.Column(scroll=ft.ScrollMode.ALWAYS) but it did not work at all. I also added and __init__ statement and took the the tasks = ft.Column to the main page and started the code via calling it from the main likewise tasks =ft.Column(scroll=ft.ScrollMode.ALWAYS),  todo.TodoApp(tasks) however this did not work either. When I use 
scroll=ft.ScrollMode.ALWAYS or ADAPTIVE with columns created within main it works but when called from within a class it did not work. I need your help in this. Thank you.

class TodoApp(ft.UserControl):
    def build(self):
        self.expand=1
        self.title_tf = ft.TextField(hint_text=vn.title_str, on_submit=self.add_clicked, width=150)
        self.definition_tf = ft.TextField(hint_text=vn.definition_str, on_submit=self.add_clicked, expand=1)
        self.tasks = ft.Column(scroll=ft.ScrollMode.ALWAYS)

"""AS YOU """

        print(self.tasks.controls)
        self.filter = ft.Tabs(
            scrollable=False,
            selected_index=0,
            on_change=self.tabs_changed,
            tabs=[ft.Tab(text=vn.all_str), ft.Tab(text=vn.active_str), ft.Tab(text=vn.completed_str)],
        )

        whole_list = sql.database_builder(what_to_do='bring', table_name=table_name, col_name_list=None,
                                          row_number=None,
                                          data_to_insert=None, id_counter_bool=None, col_type_list=None)
        count_int = 0
        if whole_list[2] is not None or whole_list[2] != []:
            for row in whole_list[2]:
                if row[8] == 0:
                    task = Task(row[2], row[3], self.task_status_change, self.task_delete, row[1])
                    self.tasks.controls.append(task)
                    count_int += 1
        self.items_left = ft.Text(str(count_int) + vn.items_title_str)

        # application's root control (i.e. "view") containing all other controls
        return ft.Column(
            width=600,
            controls=[
                ft.Row(
                    [ft.Text(value=vn.title_todo_str, style=ft.TextThemeStyle.HEADLINE_MEDIUM)],
                    alignment=ft.MainAxisAlignment.CENTER,
                ),
                ft.Row(
                    controls=[
                        self.title_tf, self.definition_tf,
                        ft.FloatingActionButton(icon=vn.add_icon_none,
                                                on_click=self.add_clicked
                                                ),
                    ],
                ),
                ft.Column(
                    spacing=25,
                    controls=[
                        self.filter,
                        self.tasks,
                        ft.Row(
                            alignment=ft.MainAxisAlignment.SPACE_BETWEEN,
                            vertical_alignment=ft.CrossAxisAlignment.CENTER,
                            controls=[
                                self.items_left,
                                ft.OutlinedButton(
                                    text=vn.delete_completed_str, on_click=self.clear_clicked
                                ),
                            ],
                        ),
                    ],
                ),
            ],
        )

**Comment by FeodorFitsner** [comment]
Could you post a runnable well-formatted example please?
